src/server/schema.js

```diff
@@ -146,12 +146,16 @@ const getAggregationType = (entry) => {
   return '';
 };
 
-const getAggregationSchemaForOneIndex = (esInstance, esIndex, esType) => {
+const getAggregationSchemaForOneIndex = (esInstance, esDict) => {
```

Comment:
minor, I'd rename this `esDict` arg to sth like `esIndividualConfig` or `esConfigElement` for readability